webapp/src/main/resources/output.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function unknownColumnWarning() {
+  // TODO: change from hardcoded to parameters
+  const params = {
+    numNotices: 1,
+    notices: [
+      {
+        filename: "hello.txt",
+        fieldName: "column name",
+        index: 5
+      }
+    ]
+  }
+  document.write(validator.templates.unknownColumnNotice(params));
```

Comment:
Did you consider using goog.soy.renderAsElement

something like: 
`return goog.soy.renderAsElement(soy.examples.helloName, params);`

(which could be later be used as `document.body.appendChild(element);`)

Reply:
No, I haven't seen this before. Looks a lot more useful than just calling the function and writing it out onto the webpage. Will change it to this.